src/bin/stratum-miner.rs

```diff
@@ -0,0 +1,343 @@
+// Copyright 2018 The Grin Developers
```

Comment:
I actually would really like to see this as as a binary for the (cuckoo-miner) [https://github.com/mimblewimble/cuckoo-miner] module rather than into Grin... this would allow people just interested in mining to just download, understand and run a much thinner mining client, as well as give us an 'official' and supported pooled mining client, something that most coins don't have. It would mean replicating some of the configuration into cuckoo-miner, but I think it would be worth it.

Reply:
I like that idea. :+1: